openlibrary/macros/FulltextSuggestionSnippet.html

```diff
@@ -19,8 +19,8 @@
           </a>&nbsp;
           <div class="fsi-snippet__quotation-mark">$_('❞')</div>
           $if page:
-          <div class="fsi-snippet__page-num" href="$ia_base_url/details/$ia">
+          <span class="fsi-snippet__page-num" href="$ia_base_url/details/$ia">
```

Comment:
also, does this need to be indented within the `if`?

Reply:
yes! I will fix that as well.